ProcessMaker/Observers/UserObserver.php

```diff
@@ -28,4 +29,15 @@ public function deleting(User $user)
             ->where('user_id', $user->id)
             ->delete();
     }
+
+    /**
+     * Handle the user "created" event.
+     */
+    public function created(User $user): void
+    {
+        $permission = Permission::byName('view-process-catalog')->first();
```

Comment:
Consider to add an upgrade to create the new permission `view-process-catalog`

The code seem to fail, when you send a string the function returns the Permission object and you do not need to call the method `first()`:

```suggestion
        $permission = Permission::byName('view-process-catalog')
```